extensions/wikia/Email/styles/specialSendEmail.scss

```diff
@@ -0,0 +1,12 @@
+.WikiaForm {
+
+	margin-bottom: 15px;
+
+	input[type="submit"] {
+		margin-top: 5px;
+	}
+
+	.input-group {
+		clear: none;
+	}
+}
```

Comment:
Yeah, I think it's overall best practise.
http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline